platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/binding/post/PostValidator.java

```diff
@@ -47,6 +48,14 @@ public void validateAuthnRequest(
         getSimpleSign()
             .validateSignature(
                 authnRequest.getSignature(), authnRequest.getDOM().getOwnerDocument());
+        if (authnRequest.getDestination() == null) {
```

Comment:
✏️ This block of code is duplicated in the `RedirectValidator`. Recommend extracting a protected method to common base class, `ValidatorImpl`.
```java
  protected void checkDestination(AuthnRequest authnRequest) throws ValidationException {
    if (authnRequest.getDestination() == null) {
      throw new ValidationException("Signed AuthnRequest must have a Destination attribute.");
    } else if (!authnRequest
        .getDestination()
        .equals(SystemBaseUrl.constructUrl("/idp/login", true))) {
      throw new ValidationException(
          "Signed AuthnRequest must have Destination attribute matching this IdP server.");
    }
  }
```